vendor-extra/JatsContentBundle/tests/EventListener/BuildView/FigGraphicFigureImageListenerTest.php

```diff
@@ -18,6 +19,29 @@ final class FigGraphicFigureImageListenerTest extends TestCase
     use ViewConvertingTestCase;
     use XmlTestCase;
 
+    /**
+     * @test
+     * @dataProvider templateChoiceProvider
+     */
+    public function it_can_choose_a_template(string $xml, ?string $expected) : void
+    {
+        $listener = new FigGraphicFigureListener($this->createFailingConverter());
```

Comment:
would expect this to be the test of `FigGraphicFigureImageListener`, departing from convention to organize tests by use case?

Reply:
Good spot.